src/components/Cart.js

```diff
@@ -0,0 +1,19 @@
+import CartItem from "./CartItem";
+
+const Cart = ({cartArray, handleRemoveFromCart}) => {
+    return (
+        <ul className="cartList wrapper">
+            <h2>Your Cart</h2>
```

Comment:
Just a small heads-up that the ul has children that aren't wrapped in <.li> tags!

Reply:
Oops! Good catch! Thank you!